app/vmalert/alerting_test.go

```diff
@@ -961,20 +962,24 @@ func TestAlertsToSend(t *testing.T) {
 }
 
 func newTestRuleWithLabels(name string, labels ...string) *AlertingRule {
-	r := newTestAlertingRule(name, 0)
+	r := newTestAlertingRule(name, 0, false)
 	r.Labels = make(map[string]string)
 	for i := 0; i < len(labels); i += 2 {
 		r.Labels[labels[i]] = labels[i+1]
 	}
 	return r
 }
 
-func newTestAlertingRule(name string, waitFor time.Duration) *AlertingRule {
-	return &AlertingRule{
+func newTestAlertingRule(name string, waitFor time.Duration, keepFiring bool) *AlertingRule {
```

Comment:
To reduce amount of changes, can we introduce a wrapper function `newTestAlertingRuleWithKeepFiring` instead?
This would allow us to use this function only in test cases where we need it.

Reply:
Sounds good, thanks!